scripts/audit_collaborators.js

```diff
@@ -26,10 +26,10 @@ module.exports = {
   },
 
   auditCollaborators: function auditCollaborators(gh, collaborators, repo) {
-    return gh.repos.getCollaborators({ repo: repo }).then(function(users) {
+    return gh.repos.getCollaborators({ repo: repo.name }).then(function(users) {
```

Comment:
Repos are now identified by `{owner, name}` pairs rather than just string name.  These calls should be updated to use the same convention, but that was out-of-scope for importing the new script.